packages/react/src/index.ts

```diff
@@ -137,8 +137,9 @@ export {default as RadioGroup} from './RadioGroup'
 export type {RelativeTimeProps} from './RelativeTime'
 export {default as RelativeTime} from './RelativeTime'
 export {SegmentedControl} from './SegmentedControl'
-export {default as Select} from './Select'
-export type {SelectProps} from './Select'
+// Curently there is a duplicate Select component at the root of the dir, so need to be explicit about exporting from the src/Select dir
+export {default as Select} from './Select/Select'
```

Comment:
@joshblack Should I explore deleting the `src/Select.tsx` file in a follow up PR?

Reply:
That would be very much so appreciated if you have the time! 🙏